src/common/components/Tooltip/Skin/index.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import React from 'react';
+import ItemHeader from '../ItemHeader';
+import Background from '../Background';
+import styles from './styles.less';
+
+type Props = {
+  data: Object,
+};
+
+const SkinTooltip = ({ data: { skin } }: Props) => {
+  return (
+    <Background>
+      <ItemHeader name={skin.name} icon={skin.icon} rarity={'white'} />
+      <span className={styles.skinTypeText}>{skin.details.type}</span>
```

Comment:
can we handle both armor and weapons?

armor:
```
https://api.guildwars2.com/v2/skins/1028
skin.details.weight_class
`${skin.details.type}${skin.type}`
```

weapons:
```
https://api.guildwars2.com/v2/skins/7756
skin.details.type
```

Reply:
Got it and added tests